src/lt/object.cljs

```diff
@@ -285,12 +287,12 @@
   [obj & r]
   (swap! obj #(apply update-in % r)))
 
-(defn assoc-in! [obj k v]
+(defn- assoc-in! [obj k v]
```

Comment:
I just used recently used this. There is a update! function (that is basically update-in, which would have been a better name btw), so assoc-in!, or assoc! would make sense for me to have at ones disposal.